ingestion/src/metadata/ingestion/source/database/dbt/metadata.py

```diff
@@ -204,21 +204,22 @@ def validate_dbt_files(self, dbt_files: DbtFiles):
                 # Validate the catalog file if it is passed
                 if dbt_files.dbt_catalog:
                     catalog_node = catalog_entities.get(key)
-                    for catalog_key, catalog_column in catalog_node.get(
-                        "columns"
-                    ).items():
-                        if all(
-                            required_catalog_key in catalog_column
-                            for required_catalog_key in required_catalog_keys
-                        ):
-                            logger.info(
-                                f"Successfully Validated DBT Column: {catalog_key}"
-                            )
-                        else:
-                            logger.warning(
-                                f"Error validating DBT Column: {catalog_key}\n"
-                                f"Please check if following keys exist for the column node: {required_catalog_keys}"
-                            )
+                    if catalog_node:
+                        for catalog_key, catalog_column in catalog_node.get(
```

Comment:
@MilanBariya can you also add a condition on `catalog_node.get("columns)`. Basically checking if the columns key is present or not

Reply:
or you can make it `catalog_node.get("columns",{})`

Reply:
@ulixius9 I add the condition and in the else part also added the warning for that.